app/components/ui/Navigation.module.scss

```diff
@@ -202,11 +204,59 @@
   }
 }
 
-.menuContainer {
+.navigationMenuContainer {
   display: flex;
   flex-direction: column;
+
+  // Is there a better way to ensure all children inherit this font-size?
+  * {
```

Comment:
probably something specific like `.navigationMenuLink a { // css here }` ?

Reply:
On that note, I'm seeing some style discrepancies from the mockup (top):
1. Font weight could be 100 or 200 lighter
2. Mockup has box shadow 
3. Nit: mockup is a little wider
4. Nit: mockup is centered and this is left aligned
5. I noticed the hover state is missing from the mockup but my guess is that the whole link should be background $gray-100 or $gray-200 on hover rather than changing text color

<img width="291" alt="image" src="https://github.com/Exygy/askdarcel-web/assets/60076088/520d0140-fa80-47d7-9f43-c05327dd972c">